suite-native/app/babel.config.js

```diff
@@ -1,4 +1,9 @@
 module.exports = {
+    env: {
+        production: {
+            plugins: ['transform-remove-console'],
+        },
+    },
     presets: ['module:metro-react-native-babel-preset'],
```

Comment:
Do we need this for exporting logs somehow? Because if not, I think it's better to keep console.logs banned in eslint and avoid them